File: IsometricShooter/Scripts/JS/GUI/StatisticGui.js
#pragma strict

var uiXP : UI.Text;
var uiKills : UI.Text;

private var statsText = "";

function OnEnable () {

	uiXP.text = "" + GlobalVars.GetXpValue();
	uiKills.text = "" + GlobalVars.GetKillCount();

}

File: IsometricShooter/Scripts/CS/GUI/StatisticGui.cs
using UnityEngine;
using UnityEngine.UI;
using System.Collections;

public class StatisticGui : MonoBehaviour {

	public Text uiXP;
	public Text uiKills;
	
	void OnEnable () {

		uiXP.text = "" + GlobalVars.GetXpValue();
		uiKills.text = "" + GlobalVars.GetKillCount();

	}
	
}
